.buildkite/nightly/pipeline.yml

```diff
@@ -62,9 +62,24 @@ steps:
           slurm_cpus_per_task: 4
           slurm_ntasks: 1
           slurm_mem: 30GB
+      
+      - label: "Coarse current AMIP: diagedmf + 0M + integrated land"
+        key: "amip_land"
+        command:
+          - echo "--- Run simulation"
+          - "julia --threads=3 --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file $CONFIG_PATH/amip_coarse_land.yml --job_id amip_coarse_land"
+        artifact_paths: "experiments/ClimaEarth/output/amip_coarse_land/artifacts/*"
+        timeout_in_minutes: 840
+        env:
+          CLIMACOMMS_DEVICE: "CUDA"
+        agents:
+          slurm_gpus_per_task: 1
+          slurm_cpus_per_task: 4
+          slurm_ntasks: 1
+          slurm_mem: 30GB
 
       - label: "Coarse current AMIP: progedmf + 0M + bucket land"
-        key: "amip_random1"
+        key: "amip_progedmf"
```

Comment:
do we need the key?

Reply:
I'm not sure. I think it's best to just include it for now